concent_api/api_testing_common.py

```diff
@@ -308,7 +333,10 @@ def execute_tests(tests_to_execute: list, objects: dict, **kwargs: Any) -> None:
 def run_tests(objects: dict, additional_arguments: Optional[dict]=None) -> None:
     if additional_arguments is None:
         additional_arguments = {}
-    (cluster_url, patterns) = parse_arguments()
+    (cluster_url, patterns, concent_1, concent_2) = parse_arguments()
+    if concent_1 is not None and concent_2 is not None:
```

Comment:
I think there should be a check that golem-messages versions are not the same.

Reply:
If they going to be the same, test will work just fine :P but I guess I could add that sanity check.

Reply:
Done